packages/sui-pde/src/index.js

```diff
@@ -2,6 +2,7 @@ export {default as PDE} from './pde.js'
 export {default as useFeature} from './hooks/useFeature.js'
 export {default as PdeContext} from './contexts/PdeContext.js'
 export {default as useExperiment} from './hooks/useExperiment.js'
+export {default as getDecision} from './hooks/common/getDecision.js'
```

Comment:
Is it a hook indeed?

Reply:
Right, I will look for a better location, thanks!

Reply:
done! @kikoruiz